index.html

```diff
@@ -17,22 +17,25 @@
                 <nav>
                     <a href="#home">Home</a>
```

Comment:
I suggest to delete this line, as header logo do this functionality

Reply:
I would instead leave both - think it depends on user habits on how return to the home view